### PR TITLE
Fixed Stylist Profile View and Added AppointmentsPage

### PR DESCRIPTION
**Pull request by anand-ts**
- Fixed merge conflicts (duplicate code and logic errors)
- Fixed Stylist Profile View
- Added AppointmentsPage.js
- Tested and worked locally

### CODE REVIEW

File: server/model/account.js [hunk 1]
```diff
@@ -16,7 +16,20 @@ const AccountSchema = new Schema({
...
+		},
+		photoContentType: {
+			type: String, // Storing the file's MIME type
+			required: false,
+		},
+		uploadedAt: {
+			type: Date,
+			default: Date.now,
+		},
+	},
```
Comment (DavidHe137) [inline server/model/account.js:32]: this file does not need to be changed, probably some issues when merging from main

**Review by DavidHe137 — CHANGES_REQUESTED** [review]
Didn't get the chance to test, but just looked over the changes. One small nit. Everything else looks great!

File: server/model/account.js (hunk 1 above)
Comment (anand-ts) [reply server/model/account.js]: Oh my bad, I reverted the changes, should be fine now.

**Review by DavidHe137 — APPROVED** [review]
Approving changes I requested previously. Have not tested, which I assume @achen443  will do.

**Pull request merged**